ads/google/imaVideo.js

```diff
@@ -425,6 +425,10 @@ export function imaVideo(global, data) {
     'background-color': 'black',
   });
   videoPlayer.setAttribute('poster', data.poster);
+  if (data.crossorigin != null) {
+    videoPlayer.setAttribute('crossorigin', data.crossorigin);
+  }
+  videoPlayer.setAttribute('poster', data.poster);
```

Comment:
Oh wait actually, isn't this the same thing as line 427? 🤔

Reply:
Perhaps the poster is just a documentation error then?

Reply:
good catch, type. fied